tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==6.0.1
+pytest-cov==2.10.1
+pytest-html==2.1.1
```

Comment:
tests also require `matplotlib`, see [.travis.yml](https://github.com/chebpy/chebpy/blob/master/.travis.yml). How do we specify the `apt` package `libfftw3-dev`? (Side note, why is that necessary? Shouldn't `pip install pyfftw` take care of the binaries, too?)